plugins/modules/site_intent.py

```diff
@@ -35,50 +35,48 @@
     choices: [ merged, deleted ]
     default: merged
   config:
-    description:
-    - List of details of site being managed.
+    description: It represents a list of details for creating/managing/deleting sites, including areas, buildings, and floors.
     type: list
     elements: dict
-    required: true
+    required: True
     suboptions:
       type:
-        description: Type of site to create/update/delete (eg area, building, floor).
+        description: Specifies the type of site operation to perform (e.g. create, update, delete).
         type: str
       site:
-        description: Site Details.
+        description: Contains details about the site being managed including areas, buildings and floors.
         type: dict
         suboptions:
           area:
-            description: Site Create's area.
+            description: Contains details for creating or managing an area within a site.
             type: dict
             suboptions:
               name:
                 description: Name of the area (eg Area1).
                 type: str
-              parentName:
-                description: Complete Parent name of the Area to be created/deleted(eg Global/).
+              parent_name:
+                description: Complete Parent name of the Area to be created/deleted(eg Global/USA).
                 type: str
           building:
-            description: Building Details.
+            description: Contains details for creating or managing a building within a site.
             type: dict
             suboptions:
               address:
                 description: Address of the building to be created.
                 type: str
               latitude:
-                description: Latitude coordinate of the building (eg 37.338).Values between -90 to +90.
-                type: int
+                description: Latitude coordinate of the building (eg 37.338). Values between -90 to +90.
               longitude:
-                description: Longitude coordinate of the building (eg -121.832).Values between -180 to +180.
+                description: Longitude coordinate of the building (eg -121.832). Values between -180 to +180.
                 type: int
               name:
                 description: Name of the building (eg building1).
                 type: str
               parent_name:
-                description: Complete Parent name of the Building to be created/deleted(eg Global/USA/San Francisco).
+                description: Complete parent name of the building to be created/deleted(eg Global/USA/San Francisco).
```

Comment:
description: Hierarchical parent path of the building, indicating its location within the site (e.g., "Global/USA/San Francisco").